inference/core/interfaces/stream/sinks.py

```diff
@@ -65,8 +68,8 @@ def render_boxes(
         video_frame (Union[VideoFrame, List[Optional[VideoFrame]]]): frame of video with its basic metadata emitted
             by `VideoSource` or list of frames from (it is possible for empty batch frames at corresponding positions
             to `predictions` list). Order is expected to match with `predictions`
-        annotator (sv.BoxAnnotator): Annotator used to draw Bounding Boxes - if custom object is not passed,
-            default is used.
+        annotator (Union[BaseAnnotator, List[BaseAnnotator]]): instance of class inheriting from supervision BaseAnnotator
```

Comment:
[comment] We are now in semantic problem - name of parameter indicates both list and single element being singular

Reply:
We already had this, i.e. `video_frame` above is already following this pattern